### DISCUSSION TITLE
Unable to use with React Native

### DISCUSSION THREAD

**Discussion by ajay-ranga** [deepgram/community#978] [General help] [answered]
If anybody have used it with react native.
Please provide a sample starter code for this.
It will be very helpfull

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Reply by ajay-ranga** [reply]
I am using STT
problem getting initial connection

**Comment by lukeocodes** [comment, marked as answer]
Hello there. Yes, our JS SDK has not yet been built with React Native in mind. I will close this because you asked it on the SDK too.
